code/campaignclassic/src/main/java/com/adobe/marketing/mobile/CampaignPushConstants.java

```diff
@@ -55,15 +55,21 @@ static final class DefaultValues {
         static final int MANUAL_CAROUSEL_MINIMUM_IMAGE_COUNT = 1;
         static final int CENTER_INDEX = 1;
         static final int FILMSTRIP_CAROUSEL_MINIMUM_IMAGE_COUNT = 3;
+        static final int ACTION_BUTTON_CAPACITY = 3;
         static final long PUSH_NOTIFICATION_IMAGE_CACHE_EXPIRY_IN_MILLISECONDS =
                 259200000; // 3 days
```

Comment:
```suggestion
        static final long PUSH_NOTIFICATION_IMAGE_CACHE_EXPIRY_IN_MILLISECONDS =
                TimeUnit.DAYS.toMillis(3);
```

Is this duration defined in the spec?

Reply:
this is not defined in the spec. my thinking behind a 3 day expiry was that push notifications tend to be pretty dynamic in their content and the possibility that a different image is used per notification is high.

Reply:
Makes sense. We can leave it as is now, but let us sync with Swarna and Steve on coming up with a value in spec/configurability. The time for which we cache directly affects the application cache load (we don't want to bloat it for an app that has very frequent push notifications). So I wonder if this should be a configurable value in the payload. But lets discuss.

Reply:
i'll add a TODO in the code to revisit this.